_includes/speakers-section.html

```diff
@@ -1,35 +1,44 @@
+	{% if site.data.speakers %}
 	<section id="speakers-section" class="speakers-section section">
 		<div class="container">
-			<h3 class="section-heading text-center mb-5">Speakers</h3>
+			<h3 class="section-heading text-center mb-5">Keynote Speakers</h3>
 			<div class="row">
 			{% for speaker in site.data.speakers %}
-
+				{% assign speaker_code = speaker[0] %}
+				{% assign speaker_data = speaker[1] %}
+				{% if speaker_data.keynote %}
```

Comment:
Please move this line before assigning the variables. You can simple check using `speaker.keynote`

Reply:
I tried this form of recording a couple of times, but, unfortunately, the data stops getting on the page.

Perhaps the reason is that instead of listing in the data file I use mapping. Perhaps the fore loop processes such data somehow differently. Or I just don’t know something :)

Reply:
The format of yml file in `speakers.yml` seems to be incorrect. You need to put a `-` at the starting of each block of data. Can you add that and update this ?

Reply:
It is not a list format.  This file uses the mapping format instead. This is neccessary to access a specific speaker. Just like in Jekyll documentation:
https://jekyllrb.com/docs/datafiles/#example-accessing-a-specific-author

Reply:
This should be a good tool to test yml file: http://www.yamllint.com/

Reply:
Looks like a valid one.

![yaml-check](https://user-images.githubusercontent.com/3881568/68014746-aed20380-fc90-11e9-930f-7c98c9389d8c.png)

Reply:
@glebkema Got that. So yaml list and hash fomat are 2 different things. Yaml list have an array like structure so each object is packed within an array element, while hashes have key, value pairs represented with an array of 2 elements. Here each data object is within a value.

Reply:
@ananyo2012  Thanks for explanation. I have not yet fully understood these terms, and your resume is very helpful.